docs/setup.md

```diff
@@ -70,13 +70,19 @@ TBD
 To run the tests on your local system in an environment that approximates how they'll be run on github, 
 cd into `tests` and run the following command (which requires the "docker compose CLI plugin" installed to work):
 ```
-   export GITHUB_REPOSITORY_OWNER=<yourname>
+   export IMGTAG=<yourname>_tests
    export USERID=<uid>
    export GROUPID=<gid>
    docker compose build
    COMPOSE_PROJECT_NAME=<yourname> docker compose run runtests
 ```
-where you replace `<uid>` and `<gid>` with your own userid and groupid; if you don't do this, the tests will run, but various pycache files will get created in your checkout owned by root, which is annoying.  `<yourname>` can be any string you want.  If you are working on a single-user machine, you can omit the `COMPOSE_PROJECT_NAME` variable; the purpose if it is to avoid colliding with other users on the same machine.
+where you replace `<uid>` and `<gid>` with your own userid and groupid; if you don't do this, the tests will run, but various pycache files will get created in your checkout owned by root, which is annoying.  `<yourname>` can be any string you want.  If you are working on a single-user machine, you can omit the `IMGTAG` and `COMPOSE_PROJECT_NAME` variables; the purpose if it is to avoid colliding with other users on the same machine.  To avoid tying this all the time, you can create a file `.env` in the `tests` subdirectory with contents:
```

Comment:
```suggestion
where you replace `<uid>` and `<gid>` with your own userid and groupid; 
if you don't do this, the tests will run, but various pycache files will get created in your checkout owned by root, which is annoying.  
`<yourname>` can be any string you want.  If you are working on a single-user machine, you can omit the `IMGTAG` and `COMPOSE_PROJECT_NAME` variables; 
the purpose of it is to avoid colliding with other users on the same machine.  
To avoid typing this all the time, you can create a file called `.env` in the `tests` subdirectory with contents:
```